posthog/api/async_migration.py

```diff
@@ -57,8 +58,17 @@ class Meta:
             "posthog_max_version",
             "posthog_min_version",
             "error_count",
+            "parameter_definitions",
         ]
 
+    def get_error_count(self, async_migration: AsyncMigration):
+        return AsyncMigrationError.objects.filter(async_migration=async_migration).count()
+
+    def get_parameter_definitions(self, async_migration: AsyncMigration):
+        definition = get_async_migration_definition(async_migration.name)
+        # Ignore typecasting logic for parameters
```

Comment:
in the future could be nice to have the inputs in the UI account for the type - but this is definitely ok now. A nice approach to look at is the greater instance settings approach (not the async migrations settings, which also take everything as a string)